io.openems.edge.aio.api/readme.adoc

```diff
@@ -0,0 +1,2 @@
+= Api for Analogue Input Output
```

Comment:
How about merging this into the `io.openems.edge.io.api` API bundle? There are already DigitalInput and DigitalOutput. Analog might fit there well.

Reply:
You're absolutely right. We applied our changes :) Thank you